pkg/utils/utils.go

```diff
@@ -84,3 +85,22 @@ func discardNil(ch chan error) chan error {
 	}()
 	return r
 }
+
+// IsWritable determines if a directory is writeable
+func IsWritable(path string) (bool, error) {
+	info, err := os.Stat(path)
+	if err != nil {
+		return false, err
+	}
+
+	if !info.IsDir() {
+		return false, errs.New("Path %s is not a directory", path)
+	}
+
+	// Check if the user bit is enabled in file permission
+	if info.Mode().Perm()&0200 == 0 {
```

Comment:
0200 i magic number to me.  What does this mean?  What if my group has permissions but not me?

I have no idea if its correct, but what the did here is at least explicit: see https://stackoverflow.com/questions/28969455/golang-properly-instantiate-os-filemode

Reply:
Sure I can define a variable or I can just leave a nice little comment of what the number means on the line above!